README.md

```diff
@@ -306,6 +301,18 @@ Terragrunt create may temporary folders for each of your terraform project based
 uniqueness_criteria = "${var.env}${var.region}/${var.project}"
 ```
 
+### Export variables to a file
+
+There are various ways to import variables such as `inputs` in the terragrunt config or `import_variables` blocks but these variables are not accesible by Terraform directly  
+To write your imported variables to a file, use the `export_variables` block. Example:  
+
+```hcl
```

Comment:
Do we export outputs of projects by default?

Reply:
Nope. You have to define a `export_variables` block, otherwise your variables will not be usable by Terraform